public/index.html

```diff
@@ -28,6 +28,7 @@
     <meta name="theme-color" content="#ffffff">
     <link href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,300;0,400;0,600;0,700;1,300;1,400;1,600;1,700&display=swap" rel="stylesheet">
 
+    <script async defer data-domain=“muda.cambiatus.io” src=“https://plausible.io/js/plausible.js”></script>
```

Comment:
the quotes here should be simple, non "tilted" `"` instead of `“`. This will cause failure on the browser's render